README.md

```diff
@@ -1 +1,71 @@
-# Ruby-capstone
\ No newline at end of file
+# Catalog of My Things
+ 
+> Imagine that you are in a catalog of a book and a game store, and you need a tool to record what books. The app that you will create will allow you to:
+
+- Add books.
+- Add games.
+- See the books.
+- See the games.
+- See the labels.
+- See the authors.
+
+And all of this will be built in a beautiful and well-organized way!
+
+## Built With
+
+- Ruby
+- Linters: [Rubocop](https://rubocop.org/)
+- Visual Studio Code
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+- A web browser like Google Chrome.
+- A code editor like Visual Studio Code with Git and Ruby.
+
+For Ruby installation.
+```
+$ ruby --version && irb
+```
+
+### Setup
+
+Clone the repository using the GitHub link provided below.
+
+### Clone
+
+In the terminal, go to your file directory and run this command.
+
+```
+$ git clone https://github.com/renercrows/Ruby-capstone.git
+```
```

Comment:
- [OPTIONAL] I suggest adding more instructions about how to run your project and test it 💯

To run the project

``` bash
$ ruby ./main.rb
```

## Testing

``` bash
$ rspec ./spec
```